### ISSUE TITLE
[Bug]: pack-mode-switcher doesn't work in server installation

### ISSUE THREAD

**Issue by EinGlasVollKakao** [ThePansmith/Monifactory#1005]
### Issue Summary

When you try to switch modes on a server, you get the following error:

![mv: cannot stat './config/server.properties': No such file or directory](https://github.com/user-attachments/assets/a49073ad-e0b5-4896-8806-eb3edf0aca21)

I think the issue occurs because of the following code:
https://github.com/ThePansmith/Monifactory/blob/984687a29b57903e6fef23c32df63597ee5eba33/pack-mode-switcher.sh#L46-L49 
It tries to copy server.properties if the current folder also has server.properties (which it does cause it's a server), but there is no server.properties file defined for any mode in [config-overrides](https://github.com/ThePansmith/Monifactory/tree/main/config-overrides). *So this logic of copying server.properties is not really needed anyway?*

### Reproduction steps

1. Setup a Monifactory server
2. Copy the pack-mode-switcher.sh file into the installation directory
3. Execute it and choose a mode

**Comment by catgirlcataclysm** [comment]
same issue here, were you able to resolve it?

**Comment by EinGlasVollKakao** [comment]
Yeah, you can just temporarily rename `server.properties` to another name, run the script and then rename it back.

**Comment by iluha168** [comment]
@Xefyr0 close this, my pr has fixed this already
